### PR TITLE
feat: add profile page

### PR DESCRIPTION
**Pull request by Aleksa013**
#### 🤔 This is a ...

- [X] New feature
- [ ] Bug fix
- [ ] Performance optimization
- [ ] Refactoring
- [ ] Test Case
- [ ] Documentation update
- [ ] Other

#### 💡 Background and solution
Implement Profile page
_Describe the big picture of your changes here_
pass
mail
name
#### 🎯 Related issues

_Links to related issues_
https://angularteam.atlassian.net/jira/software/projects/SCRUM/boards/1/backlog?selectedIssue=SCRUM-7

### CODE REVIEW

File: src/app/pages/profile-page/profile-page.component.ts [hunk 1]
```diff
@@ -40,16 +61,30 @@ export class ProfilePageComponent implements OnInit {
...
+    this.visibleModal = true;
+  }
+
   public updateUserData(newName: string, newEmail: string): void {
     this.store.dispatch(
       updateUserName({
-        name: newName.length ? newName : this.name,
-        email: newEmail.length ? newEmail : this.email,
+        name: newName.length ? newName : this.currentName,
+        email: newEmail.length ? newEmail : this.currentEmail,
```
Comment (NataRep) [inline src/app/pages/profile-page/profile-page.component.ts]: It seems to me that this approach may lead to the user entering spaces as a name. 
But maybe I'm wrong and trim is applied to fields somewhere else

**Review by NataRep — APPROVED** [review]

**Review by Arthurchkeee — APPROVED** [review]
Good job!

**Review by Arthurchkeee — APPROVED** [review]

File: src/app/components/header/header.component.ts [hunk 2]
```diff
@@ -19,6 +19,9 @@ export class HeaderComponent implements OnInit {
 
   ngOnInit(): void {
     this.role$ = this.store.select(selectAccess);
+    if (localStorage.getItem('token')) {
+      this.store.dispatch(getUser());
+    }
   }
```
Comment (NataRep) [inline src/app/components/header/header.component.ts]: please move this to app.component from header

**Review by NataRep — CHANGES_REQUESTED** [review]

File: src/app/components/header/header.component.ts (hunk 2 above)
Comment (Aleksa013) [reply src/app/components/header/header.component.ts]: done

File: src/app/pages/profile-page/profile-page.component.ts (hunk 1 above)
Comment (Aleksa013) [reply src/app/pages/profile-page/profile-page.component.ts]: thank you!
I didn't think about it

Comment (Aleksa013) [reply src/app/pages/profile-page/profile-page.component.ts]: done

**Review by mogitrash — APPROVED** [review]
Good job! 🥇

**Pull request merged**